test/e2e/kubevirt.go

```diff
@@ -1558,11 +1653,15 @@ runcmd:
 			description string
 			resource    resourceCommand
 			test        testCommand
-			topology    string
-			role        string
+			topology    udnv1.NetworkTopology
+			role        udnv1.NetworkRole
+			ingress     string
```

Comment:
maybe add a new enum type for the ingress types ? Or just `routed` since I assume you have `routed` and empty ingress.

Reply:
I don't think is it worth it, there is just one "routed" test